### PR TITLE
add to changelog with new monitoring apis

### PR DESCRIPTION
**Pull request by frivieremicroej**

### CODE REVIEW

File: VEEPortingGuide/architectureChangelog.rst [hunk 1]
```diff
@@ -49,6 +49,16 @@ Core Engine
...
 
 
+Core Engine
+~~~~~~~~~~~
+
+- [Multi] - Added new functions to Low Level API ``LLMJVM_MONITOR_impl.h`` for CPU Control monitoring
+
+   -  ``void LLMJVM_MONITOR_IMPL_on_quota_reset(int32_t context_id, int32_t quota)``: called by the Core Engine when a quota is reset to an execution context..
+   -  ``void LLMJVM_MONITOR_IMPL_on_quota_reached(int32_t context_id)``: called by the Core Engine the quota of the given execution context has been reached..
+   -  ``void LLMJVM_MONITOR_IMPL_on_quantum_counters_reset()``: called by the Core Engine when the quantum computation is reset.
```
Comment (jerome-leroux) [inline VEEPortingGuide/architectureChangelog.rst:59]: I am note sure:
```suggestion
   -  ``void LLMJVM_MONITOR_IMPL_on_quota_reset(int32_t context_id, int32_t quota)``: called by the Core Engine when a quota is reset for an execution context.
   -  ``void LLMJVM_MONITOR_IMPL_on_quota_reached(int32_t context_id)``: called by the Core Engine the quota of the given execution context has been reached.
   -  ``void LLMJVM_MONITOR_IMPL_on_quantum_counters_reset()``: called by the Core Engine when all the quantum counters are reset.
```

Should we review the changelog entry for `LLKERNEL_quantaConsumed(int32_t quanta)`:
```
- [Multi] - Added function `LLKERNEL_quantaConsumed(int32_t quanta)` in `LLKERNEL.h` to allow a native function to increase the quantum counter for CPU control.
```

**Review by jerome-leroux — CHANGES_REQUESTED** [review]

File: VEEPortingGuide/architectureChangelog.rst (hunk 1 above)
Comment (frivieremicroej) [reply VEEPortingGuide/architectureChangelog.rst]: updated, see new commits

**Comment by frivieremicroej** [comment]
once stabilized, TODO report changes to the original header file

**Review by frivieremicroej — COMMENTED** [review]
once stabilized, TODO report changes to the original header file

File: VEEPortingGuide/architectureChangelog.rst [hunk 2]
```diff
@@ -45,9 +45,14 @@ Core Engine
 ~~~~~~~~~~~
 
 - [Multi] - Increased execution quota precision
-- [Multi] - Added function `LLKERNEL_quantaConsumed(int32_t quanta)` in `LLKERNEL.h` to allow a native function to increase the execution counter.
+- [Multi] - Added function `LLKERNEL_quantaConsumed(int32_t quanta)` in `LLKERNEL.h`, allowing a native function to increment the quantum counter of the current execution context.
 - [Multi] - Fixed watchdog which prevents the Core Engine to stop because of a pending thread.
+- [Multi] - Added new functions to Low Level API ``LLMJVM_MONITOR_impl.h`` for CPU Control monitoring
 
+   -  ``void LLMJVM_MONITOR_IMPL_on_quota_reset(int32_t context_id, int32_t quota)``: called by the Core Engine when the quota for an execution context is updated.
+   -  ``void LLMJVM_MONITOR_IMPL_on_quota_reached(int32_t context_id)``: called by the Core Engine the quota for an execution context has been reached.
```
Comment (jerome-leroux) [inline VEEPortingGuide/architectureChangelog.rst:53]:
```suggestion
   -  ``void LLMJVM_MONITOR_IMPL_on_quota_reached(int32_t context_id)``: called by the Core Engine when the quota for an execution context has been reached.
```

**Pull request merged**